internal/healthchecks/network_check.go

```diff
@@ -103,8 +107,16 @@ func (c NetworkCheck) Name() string {
 
 func (c NetworkCheck) RunCheck(logger logs.StructuredLogger) error {
 	var networkErrors []error
-	for _, r := range requests {
+	ctx := context.TODO()
+	p := platform.FromContext(ctx)
+	for _, r := range commonRequests {
 		networkErrors = append(networkErrors, r.SendRequest(logger))
 	}
+	if p.ResourceOverride == nil {
```

Comment:
@alex-basinov Note that this will need to be fixed before merging back into `master`, since GCE checks will also need to run if the resource is explicitly specified as `"gce_instance"`.

Reply:
Actually, it's done as part of #1539, so we can pick this up on the next merge.